binary/private/binary_mdot.f90

```diff
@@ -735,7 +735,7 @@ end subroutine get_info_for_ritter
       real(dp) function calculate_kolb_mdot_thick(b, indexR, rl_d) result(mdot_thick)
          real(dp), intent(in) :: rl_d
          integer, intent(in) :: indexR
-         real(dp) :: F1, F3, G1, dP, q, rho, p, grav, hp, v_th, rl3, q_temp
+         real(dp) :: F1, F3, G1, d_P, q, q_temp
```

Comment:
It's best to change this from `dP` to `d_P`. I'm not sure how the (case-insensitive) Fortran compiler handles with with `dp` also being defined